test/ElectrumClientTest.js

```diff
@@ -1,20 +1,18 @@
-/*
-const ElectrumClient = require("../src/ElectrumClient")
-const fs = require("fs")
-const chai = require("chai")
-const assert = chai.assert
-const config = require("../../../src/config/config.json")
+import Client, { scriptToHash } from "../src/lib/ElectrumClient.js"
```

Comment:
Could we rename this as `import Client as ElectrumClient, { scrip...`? I think it'll be clearer. Or we could change the exported class name in `ElectrumClient.js`, which would be even better IMO.

Reply:
I opted for simply importing the class with the name `ElectrumClient` (in https://github.com/keep-network/tbtc.js/pull/133/commits/75fa11240670cca56ca39e566d4b5a472e2115a1). There is a third party library class that is named `ElectrumClient` in `ElectrumClient.js` and adding another class with that name there could cause confusion.